domino/constants.py

```diff
@@ -0,0 +1,5 @@
+"""
+Minimum Domino version supported by this python-domino library
+"""
+
+MINIMUM_SUPPORTED_DOMINO_VERSION = '4.0.0'
```

Comment:
Should it be '4.2' instead? Not sure if '4.2.0' or what we actually have, `cloud-develop` reports 4.2.0-SNAPSHOT